### ISSUE TITLE
2020 Spring Fest ATX Mobility Guide Web Map / App - T. Whitson

### ISSUE THREAD

**Issue by johnclary** [cityofaustin/atd-geospatial#600]
@jaime-mckeown plz add details
- The PIO has requested a Web Map to replace the static map that was normally created by the Graphic Designer for this year's Spring Festival.
- I will work with Tyler to create this Web Map / App, need to add requirements.

**Comment by jaime-mckeown** [comment]
Started working on the Web Map, added:
- Metro Rail and Stations
- Metro Rapid and Stops
- Pedicab staging areas
- Bicycle Route
- Ann and Roy Butler Hike and Bike Trail
- Created, Published and added Downtown Public Restroooms

**Comment by jaime-mckeown** [comment]
- Created a Landmarks layer for the Convention Center so it's easy for users to see.
- Ended up staying with Austin Bike Map feature layer for routes and trails. The urban trail data that is shared publicly is not useful.
- Had to create new MetroRail layer to modify the end of the line due to construction at the convention center for the new downtown station. Added stations to this layer as the layer I copied in AGOL was requiring a token to use.
- Created and published passenger dropoff/pickup zones and added to the Map.
- Created Web App and shared so PIO can give feedback. We are going to use the ESRI Street Map as it's more detailed then the COA street map.
